app/models/user.rb

```diff
@@ -62,6 +64,9 @@ class User < ApplicationRecord
     dependent: :delete_all,
     inverse_of: :owner
 
+  attribute :quota_use_site_default, :boolean, default: true
```

Comment:
could this be simplified by just having the user quota field set to nil by default, which means "use site default"? Then this boolean wouldn't be needed?

Reply:
I've put this comment in the wrong place really. If you change the migration line to: `add_column :users, :quota, :integer, null: true`, it should then allow nil values.

Reply:
Yeah, I was thinking that `quota: nil` could mean "use site policy", but you've put more thought into this than me :)